app/src/main/res/layout/profile_chooser_fragment.xml

```diff
@@ -106,27 +105,26 @@
       app:layout_constraintEnd_toEndOf="parent">
 
       <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="24dp"
-        android:fontFamily="sans-serif"
+        android:id="@+id/textView"
+        style="@style/Subtitle1"
         android:gravity="center"
         android:minHeight="48dp"
         android:text="@string/profile_chooser_administrator_controls"
         android:textColor="@color/profileChooserGreyTextColor"
-        android:textSize="16sp" />
+        app:layout_constraintEnd_toStartOf="@+id/profile_chooser_setting_icon"
+        app:layout_constraintTop_toTopOf="parent" />
 
       <ImageView
         android:id="@+id/profile_chooser_setting_icon"
         android:layout_width="48dp"
         android:layout_height="48dp"
-        android:layout_marginEnd="4dp"
-        android:layout_marginBottom="24dp"
```

Comment:
Why is this margin removed? 

If you notice both the this image the bottom text and icon have a different alignment as compared to earlier case and thats because of change in margin/padding in this and above view.

left - Your Changes
right - Previous UI

<img width="1029" alt="Screenshot 2020-10-13 at 12 19 49 PM" src="https://user-images.githubusercontent.com/9396084/95825505-9696af80-0d4e-11eb-99c5-b63dad0b26fe.png">

Reply:
Done